frontend/stubs/functions.py

```diff
@@ -1,12 +1,20 @@
 """Stub file for built in functions"""
+from typing import TypeVar, List, Tuple, Dict
 
+Any = TypeVar("Any")
```

Comment:
Here you're using these not really as type variables, but as union types, right? That's... a bit hacky, but interesting. Fine by me I think...